TAF/utils/scripts/snap/deploy-device-service.sh

```diff
@@ -1 +1,3 @@
-#!/bin/sh
+#!/bin/bash
+
+logger "[snap] deploy-device-service: $@"
```

Comment:
It seems like this script is called by TAF and that all it does is write a log entry based upon the input args (i.e. the service name)? If so, a comment or two explaining that would be helpful.

Reply:
This script is not actually used - and I've removed it.